pkg/monitoring/deployment_test.go

```diff
@@ -20,11 +24,122 @@ func (mock *MockDeployment) DeleteDeployment(namespace, name string) error {
 	return nil
 }
 
-func TestDeletePomPusher(t *testing.T) {
+// TestCreateAppServiceAccounts tests the creation of a VMI system deployment object.
+// GIVEN a set of values, such as secrets and labels
+//  WHEN I call GetSystemDeployments
+//  THEN there should be a singe deployment object created for the VMI system binding
+//   AND that object should have the expected values
+func TestCreateDeployment(t *testing.T) {
```

Comment:
This function is called TestCreateDeployment but never actually calls CreateDeployment - I'm confused.

Reply:
bad comment on my part